.github/ISSUE_TEMPLATE.md

```diff
@@ -1,15 +1,18 @@
-* [ ] 'Smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, your usage of Mocha, or Mocha itself
+* [ ] Checked that your issue isn't already filed by searching for relevant keywords in the [issue tracker](https://github.com/mochajs/mocha/issues) and cross-referencing [issues with the `common mistake` label](https://github.com/mochajs/mocha/issues?utf8=%E2%9C%93&q=is%3Aissue%20label%3Acommon-mistake%20) <!-- NOTE: in the event that you find a relevant existing issue but its solution does not work for you, it is possible that you have a different although similar issue; in such cases opening a new issue is appropriate. ***However***, in the event you do wish to add to an existing issue (e.g. because it has no solution yet), **we highly recommend following the same steps and instructions as in reporting a new issue**, to minimize comments troubleshooting your particular case on someone else's issue. If you don't have new information to provide, just hit the "thumbs-up" reaction on the issue; we can look at those numbers to get an idea of how many people are affected without spam in the comments. -->
+* [ ] 'Smoke tested' the code to be tested by running it outside the real test suite to get a better sense of whether the problem is in the code under test, your usage of Mocha, or Mocha itself. If using any additional tooling make sure to use the exact same toolset outside Mocha (see below).
+* [ ] Checked next-gen ES issues and syntax problems (if any) by using ***the same*** *environment and/or transpiler configuration* ***without Mocha*** to ensure it isn't just a feature that actually isn't supported in the environment in question or a bug in your code. For example:
+    * Transpilers often do nothing to indicate that they ran OK but didn't change the code you expected them to change; if the problem appears when using a transpiler and Mocha, it usually is still there if you use just the transpiler, but you have to actually run the transpiled code to tell (since the transpiler doesn't know that what it transpiled didn't include what you wanted it to include).
+    * Using multiple sets of tools together, such as a transpiler, a bundler and a browser, can make it inobvious which tool isn't doing its part (e.g. a bundler or a browser might be handling syntax you thought you transpiled, when in fact the transpiler is not working), especially if the tests do not use the entire same toolset and environment.
 * [ ] Ensured that there is no discrepancy between the locally and globally installed versions of Mocha. You can find them with:
 `node node_modules/.bin/mocha --version`(Local) and `mocha --version`(Global). We recommend avoiding the use of globally installed Mocha.
```

Comment:
We could also add to the prerequisites list a link to a new wiki page "Common Issues". This could be in addition to the entries already here, or in place of some of the details about things like transpilers. I wouldn't want it to replace the recommendation to search the issue tracker in various ways; but it might be best to link to the "Common Issues" page before the issue tracker since it would provide more straightforward fixes *and* more detailed explanations with less noise. That page could have the following format:

# Common Issues

## How to fix

### Name of Issue

- examples: "error message here"
- fix: do x instead of y
- more info: [link to corresponding issue in details section]

### Name of another issue

same type of list here

## Details (or "Why?")

### Name of Issue

This is an issue that occurs when you foo the bar. Bar expects to be fizzed or buzzed rather than foo'd, so it throws an error. You can manually fizz and buzz the bar instead, but it's easier and more reliable to tell Mocha you're using a bar and let Mocha handle the fizzing and buzzing.

### Name of Other Issue

This is an issue with bundling and transpiling at the same time and then running against a browser vs. the current version of Node only partly supporting New Idea Voodoo. As a language feature there's nothing Mocha can do to implement it (and it should not anyway, since your tests should indicate whether the code will work in environments that don't have the test runner "helping" them), and nothing Mocha can do to interfere with it when it is implemented either. However, the implementations available at this point can be very complex due to the interaction between transpilers and bundlers and the differences between how browsers have done it and how Node has done it. We recommend consulting the documentation and support for these projects, where you can get more detailed and up-to-date info and better assistance.

Reply:
(Tangentially, this would be the sort of wiki page that might be better off as part of our new docs collection once we split up the site into multiple pages of documentation.)

Reply:
this is pretty much the definition of a FAQ.